source/installation-guide/more-installation-alternatives/wazuh-from-sources/wazuh-agent/index.rst

```diff
@@ -85,13 +85,12 @@ Installing Wazuh agent from sources
     .. raw:: html
 
        <h2>Uninstall</h2>
-    
-    To uninstall Wazuh agent:
+
+    To uninstall Wazuh agent, set the right installation path for ``WAZUH_HOME``:
```

Comment:
I prefer something like `set WAZUH_HOME with the current installation path`

Reply:
Updated!